Meadowlark-js/lerna.json

```diff
@@ -3,12 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "v0.3.0-pre-35",
+  "version": "0.4.0-pre.0",
   "npmClient": "npm",
-  "useWorkspaces": true,
-  "command": {
```

Comment:
This could potentially allow users to create tags with lerna from branches, but this is currently possible from git tags also